tool/src/org/antlr/v4/codegen/Target.java

```diff
@@ -457,7 +457,10 @@ public String getBaseVisitorFileName(boolean header) {
 	 * @return the serialized ATN segment limit
 	 */
 	public int getSerializedATNSegmentLimit() {
-		return Integer.MAX_VALUE;
+		// Make strings encoding ATN fit on a single line conveniently so
+		// we can march through generated code more easily.  Only Java has
+		// an actual string len limit (16 bits).
+		return 80;
```

Comment:
Maybe it's better to use `120` instead of `80`, especially for uninformative data (for user). `80` is old-school and it's a bit outdated line length restriction, modern monitors support much more characters. By the way, IntelliJ has `120` vertical rulers by default.